webapp/channels/src/components/advanced_text_editor/remote_user_hour.tsx

```diff
@@ -88,27 +88,33 @@ const RemoteUserHour = ({teammateId, displayName}: Props) => {
     }
 
     return (
-        <Container>
-            <Icon/>
-            <FormattedMessage
-                id='advanced_text_editor.remote_user_hour'
-                defaultMessage='The time for {user} is {time}'
-                values={{
-                    user: displayName,
-                    time: (
-                        <Timestamp
-                            useRelative={false}
-                            value={timestamp}
-                            useDate={false}
-                            userTimezone={teammateTimezone}
-                            useTime={{
-                                hour: 'numeric',
-                                minute: 'numeric',
-                            }}
-                        />
-                    ),
-                }}
-            />
+        <Container className='RemoteUserHour'>
+            <Icon className='icon moonIcon'/>
+            <span className='message'>
```

Comment:
I believe `FormattedMessage` already makes a `span`, is there a reason we have a second one wrapped around?

Reply:
yeah it wasn't needed. Removed.